lib/alerts/cli.ts

```diff
@@ -295,9 +299,10 @@ class AlertsCli {
       updated.targets = [];
     }
 
-    const id = this.alertIdFromArgv(argv);
```

Comment:
Bug fix: `alertIdFromArgv` is not defined. What was likely meant here was `alertIdFromArgs` (note the "s" vs "v" at the end). Git blame shows this has been broken for many years (~5, IIRC).